modules/httpAPI/dashboard/index.html

```diff
@@ -1,89 +1,41 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <meta charset="UTF-8">
     <title>FlapAlertedPro</title>
-    <style>
-        #prefixTable {
-            width: 100%;
-            font-family: Arial, Helvetica, sans-serif;
-            border-collapse: collapse;
-        }
-
-        #prefixTable td,
-        #prefixTable th {
-            border: 1px solid #ddd;
-            padding: 8px;
-        }
-
-        #prefixTable tr:nth-child(even) {
-            background-color: #f2f2f2;
-        }
-
-        #prefixTable tr:hover {
-            background-color: #ddd;
-        }
-
-        #prefixTable th {
-            padding-top: 12px;
-            padding-bottom: 12px;
-            text-align: left;
-            background-color: #c04c09;
-            color: white;
-        }
-
-        .headerDiv {
-            height: 16em !important;
-            display: inline-block;
-            width: 50%;
-            box-sizing: border-box;
-        }
-
-        #wrapper {
-            box-sizing: border-box;
-        }
-
-        .disabledLink {
-            color: currentColor;
-            cursor: not-allowed;
-            opacity: 0.5;
-            pointer-events: none;
-            text-decoration: none;
-        }
-    </style>
+    <link rel="stylesheet" href="assets/css/style.css" integrity="sha384-M7r2/qL5kDteUHazfjvlljZg5t9R8ISiH1wlTsIp/vkHkSTQnLyDqAj0RIYeDcUQ">
 </head>
 
 <body>
-    <h2 style="margin-bottom:0.2em">FlapAlertedPro by Kioubit</h2>
-    <h3 id="dataInfo" style="margin-top:0">Live Data</h3>
-    <div id="connectionLost" style="display: none;">
-        <p style="text-align:center;color:red; font-size:2em">Connection lost</p>
+    <h2 id="mainHeading">FlapAlertedPro by Kioubit</h2>
+    <h3 id="dataInfo">Live Data</h3>
+    <div id="connectionLost">
+        <p id="connectLostText">Connection lost</p>
     </div>
     <div>
-        <div style="height: 16em; padding-bottom: 16em;" id="wrapper">
+        <div id="wrapper">
             <div class="headerDiv">
-                <div style="height: 16em; overflow: hidden;" id="justgage"></div>
+                <div id="justgage"></div>
             </div>
-            <div style="float: right;" class="headerDiv">
-                <div style="height: 16em; overflow: auto;" id="prefixTableBox"></div>
+            <div class="headerDiv floatRight">
+                <div id="prefixTableBox"></div>
             </div>
         </div>
-        <div style="clear: both; padding-bottom: 1em;"></div>
+        <div></div>
```

Comment:
previous css styles get lost here

Reply:
Oh, seems like I oversee it.

Reply:
should be fixed in https://github.com/Kioubit/FlapAlertedPro/pull/3/commits/639284f0d9c25d342e5a1a3d730c4327fcab1647